.github/workflows/publish-python.yml

```diff
@@ -0,0 +1,153 @@
+name: Publish Python Bindings
+on:
+  workflow_call:
+    inputs:
+      ref:
+        description: 'commit/tag/branch reference'
+        required: true
+        type: string
+      package-version:
+        description: 'version for the python package (MAJOR.MINOR.BUILD)'
+        required: true
+        type: string
+      publish:
+        description: 'value indicating whether to publish to pypi.'
+        required: true
+        type: boolean
+        default: false
+    secrets:
+      PYPI_API_TOKEN:
+        description: 'api token to authenticate to pypi'
+        required: true
+
+jobs:
+  build-macos-wheels:
+    runs-on: macos-latest
+    strategy:
+      matrix:
+        python: ["3.8", "3.9", "3.10"]
+    steps:
+      - name: Checkout breez-sdk repo
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ inputs.ref || github.sha }}
+
+      - name: "Install Python"
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python }}
+
+      - uses: actions/download-artifact@v3
+        with:
+          name: sdk-bindings-darwin-universal
+          path: libs/sdk-bindings/bindings-python/src/breez_sdk
+
+      - uses: actions/download-artifact@v3
+        with:
+          name: bindings-python
+          path: libs/sdk-bindings/bindings-python/src/breez_sdk
+
+      - name: Clean up downloaded files
+        run: |
+          rm -f libs/sdk-bindings/bindings-python/src/breez_sdk/*.a
+          ls -R libs/sdk-bindings/bindings-python
```

Comment:
This step does not exist on linux. What does it do? Can we do that on uploading the artifact instead?

Reply:
The build step for macOS outputs the bindings as `.dylib` as well as a `.a` file. Python expects the `.dylib` and fails if the `.a` is present, thus the removal.

We could probably do that on the uploading as well although I'm not 100% certain on what the publishing for other languages steps need. Do you know if there's maybe one that relies on the `.a` file?

Reply:
I have no idea tbh. This looks fine to me.